sdk/anomalydetector/azure-ai-anomalydetector/src/main/java/com/azure/ai/anomalydetector/AnomalyDetectorClient.java

```diff
@@ -56,13 +100,16 @@ public final class AnomalyDetectorClient {
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
      * @return the response of entire anomaly detection.
      */
+    @Generated
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public EntireDetectResponse detectEntireSeries(DetectRequest body) {
-        return this.serviceClient.detectEntireSeries(body);
+    public EntireDetectResponse detectEntireSeriesSync(DetectRequest body) {
```

Comment:
@alzimmermsft should it only be updating the service impl calling method here and not the exposed public API?

Reply:
This looks like naming when using `sync-stack` needs to be revisited slightly to prevent the API from changing.

Reply:
https://github.com/Azure/autorest.java/pull/1811